### ISSUE TITLE
Enhancement: Changes To Vocal Track Muting On Miss

### ISSUE THREAD

**Issue by Average-FNF-Modder** [FunkinCrew/Funkin#3950]
### Issue Checklist

- [x] I have properly named my enhancement
- [x] I have checked the Issues/Discussions pages to see if my enhancement has already been suggested

### What is your suggestion, and why should it be implemented?

I'm here to suggest 2 modifications to the way the muting of the vocal track when the player is missing is handled.
First, I would suggest having ghost misses no longer mute the track.  There's no reason pressing an extra note should mute the track when you still hit all the singing notes in my eyes. The second change I would make is to have regular note miss mutes go away after sometime, in my opinion like 1 single beat, of course with that timer getting reset every time you miss again. These changes make a couple things nicer. For one thing, if the player only has one vocal track instead of separated vocals, it'll stop ghost misses or a miss on the final note of a set from completely muting the opponent's side till the player gets to hit a note again. Even with separated vocals, there may be flairs like Pico's burping that get played without a note, that ghost misses and misses of the previous note could fuck up for no reason.

**Comment by Hundrec** [comment]
https://github.com/FunkinCrew/Funkin/pull/3012 removes ghost muting.

For your second suggestion, I'm concerned that could cause a side effect where Boyfriend beeps a tiny bit before missing again, negating the muting of his vocals. Perhaps this could be done if the game detects only a single vocal track.

**Comment by Average-FNF-Modder** [comment]
> [#3012](https://github.com/FunkinCrew/Funkin/pull/3012) removes ghost muting.
> 
> For your second suggestion, I'm concerned that could cause a side effect where Boyfriend beeps a tiny bit before missing again, negating the muting of his vocals. Perhaps this could be done if the game detects only a single vocal track.

I believe this wouldn't happen as every miss resets the timer, so unless there's a super long turn break, in which case I say it's fair game to allow a cut off beep to play again like happens with first misses anyways, this shouldn't be a problem.

**Comment by Hundrec** [comment]
Good point, but the timer would have to be delicately balanced to avoid those excess beeps while allowing a potential one-track opponent to sing. 
Set the timer too short and BF sings without inputs
Too long and the opponent would be muted for a good while anyway
